cohorts/variant_stats.py

```diff
@@ -1,14 +1,12 @@
 from collections import namedtuple
-
+import numpy as np
```

Comment:
This is leftover from the earlier version of the divide-by-zero issue, which did use np. I'll remove it.